CI/macos/appcast_convert.py

```diff
@@ -0,0 +1,83 @@
+import os
+
+import xmltodict
```

Comment:
Is there some Python standard/practice for spacing these apart?

Reply:
Yes, see PEP8 section for imports: https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.